webpack.config.js

```diff
@@ -66,7 +66,9 @@ const config = {
     modules: ['node_modules'],
     // Required for some module configs which use these fields
     // See https://github.com/flatiron/director/issues/349
-    mainFields: ['browserify', 'browser', 'module', 'main']
+    mainFields: ['browserify', 'browser', 'module', 'main'],
+
+    symlinks: false
```

Comment:
just curious why was this needed?

Reply:
This allows us to symlink our skin into the `skins` directory:

```
$ ls -ls static_src/skins/
total 8
0 drwxr-xr-x  3 me  staff  102 13 Oct 09:09 cg
8 lrwxr-xr-x  1 me  staff   56  5 Oct 14:52 govau -> /Users/me/go/src/github.com/govau/ci-console/skin/
```

And we are symlinking (at the very least) for development: our skin is in another repo and I symlink it up.

Without this change, the UI won't build, giving the error:

```
ERROR in /Users/me/go/src/github.com/govau/ci-console/skin/index.js
Module build failed: ReferenceError: Unknown plugin "transform-object-rest-spread" specified in "base" at 0, attempted to resolve relative to "/Users/me/go/src/github.com/govau/ci-console/skin"
<omitted>
```

Upgrading babel and friends could fix this, but this config change does fix it and doesn't appear to affect anything else.

Reply:
thanks for that! @jonathaningram could you make an issue to revisit this when babel is upgraded?